src/network/clientpackethandler.cpp

```diff
@@ -587,6 +588,37 @@ void Client::handleCommand_MovePlayer(NetworkPacket* pkt)
 	m_ignore_damage_timer = 3.0;
 }
 
+void Client::handleCommand_KnockBack(NetworkPacket *pkt)
+{
+	LocalPlayer *player = m_env.getLocalPlayer();
+	assert(player != NULL);
+
+	v3f direction;
+	f32 time_knockback;
+	u16 id_player_knockback;
+	std::string playername;
+
+	*pkt >> id_player_knockback >> playername >> direction >> time_knockback;
+
+	player->got_teleported = true;
+	if (strcmp(player->getName(), playername.c_str()) == 0) {
```

Comment:
@sfan5 there is client animations on local Player CAO, not only current player but all players :)